likecoin/admin/restful.php

```diff
@@ -334,7 +345,8 @@ function likecoin_get_iscn_full_info( $request ) {
 		$iscn_full_info['iscnHash']    = $iscn_info['iscn_hash'];
 		$iscn_full_info['iscnId']      = $iscn_info['iscn_id'];
 		$timezone                      = wp_timezone_string();
-		$iscn_full_info['iscnVersion'] = '1 (' . $iscn_info['last_saved_time'] . ' ' . $timezone . ')';
+		$iscn_timestamp_local_time     = get_date_from_gmt( gmdate( 'Y-m-d H:i:s', $iscn_info['last_saved_time'] ), 'Y-m-d H:i:s' );
+		$iscn_full_info['iscnVersion'] = '1 (' . $iscn_timestamp_local_time . ' ' . $timezone . ')';
```

Comment:
Please add a comment for iscnVersion should not be hardcoded as `1`, but should fetch from chain iscn api

Reply:
Also I think for an api, we should not format the string here, but return raw timestamp and version i.e. `1`, and let frontend format the string to display format as it desired.